person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative './nameable'
+require_relative('./rental')
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] to make the associations work Getters and Setters for @rentals should be present.